tests/ChatClient.hs

```diff
@@ -117,7 +117,9 @@ testCoreOpts =
         { dbConnstr = testDBConnstr,
           -- dbSchemaPrefix is not used in tests (except bot tests where it's redefined),
           -- instead different schema prefix is passed per client so that single test database is used
-          dbSchemaPrefix = ""
+          dbSchemaPrefix = "",
+          dbPoolSize = 3,
```

Comment:
why make it different from one used in chat?

Reply:
different how? In chat it's configured via option. Here we test that pool works.